multinode_installation/Vagrantfile

```diff
@@ -11,5 +11,7 @@ Vagrant.configure("2") do |config|
 
 	config.vm.provision :ansible do |ansible|
 		ansible.playbook = "playbook.yml"
+    # https://gist.github.com/phantomwhale/9657134
+    ansible.raw_arguments = Shellwords.shellsplit(ENV['ANSIBLE_ARGS']) if ENV['ANSIBLE_ARGS']
 	end
```

Comment:
If you run
```ANSIBLE_ARGS='--tags=develop' vagrant provision ```
this will only run [ansible tasks that are tagged](http://docs.ansible.com/ansible/playbooks_tags.html) 'develop' on your local vm.